url.bs

```diff
@@ -3102,6 +3102,23 @@ let params = new URLSearchParams({key: "730d67"})
 params.toString() // "key=730d67"</code></pre>
 </div>
 
+<div class=note>
+ <p>As {{URLSearchParams}} objects use the <a><code>application/x-www-form-urlencoded</code></a>
+ format underneath there are some difference with how it encodes certain code points compared to
+ {{URL/search}}.
```

Comment:
and href

Reply:
maybe just say URL then?

Reply:
True, although I think explicitly comparing the `searchParams` API to the `href` and `search` APIs is helpful. But if there are examples then it's less important.